apps/chess-training-app/Dockerfile.dev

```
@@ -0,0 +1,22 @@
+FROM node:22-alpine
+
+RUN apk add --no-cache bash git openssl
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)